graylog2-web-interface/src/components/users/TokenList.test.tsx

```diff
@@ -64,8 +64,8 @@ describe('<TokenList />', () => {
     const nameInput = await screen.findByPlaceholderText('What is this token for?');
     userEvent.type(nameInput, 'hans');
 
-    const ttlInput = await screen.findByPlaceholderText("What is this token's TTL?");
-    userEvent.type(ttlInput, 'PT72H');
+    const ttlInput = await screen.findByLabelText("Token TTL");
+    fireEvent.change(ttlInput, {target: {value: 'PT72H'}});
```

Comment:
@ousmaneo I used the same 'fireEvent.change' call from the enterprise test we fixed and got this passing, but I would appreciate a double check.